development/kyma-slack-connector/releaseClusterCreatedFunction/handler.py

```diff
@@ -0,0 +1,85 @@
+import base64
+import json
+import os
+
+from slack_bolt import App
+from slack_sdk.errors import SlackApiError
+
+def main(event, context):
+	# Using SLACK_BOT_TOKEN environment variable
+	app = App(
+	)
+	slack_api_id = os.environ['SLACK_API_ID'].replace('-', '_')
+	slack_channel = os.environ['NOTIFICATION_SLACK_CHANNEL']
+	env_prefix = os.environ['ENV_PREFIX']
+	base_url = os.environ['{}_SLACK_CONNECTOR_{}_GATEWAY_URL'.format(env_prefix, slack_api_id)]
+	# Set Slack API base URL to the URL of slack-connector application gateway.
+	app.client.base_url = "{}/".format(base_url)
+	print("received message with id: {}".format(event["data"]["ID"]))
+	print("Slack api base URL: {}".format(app.client.base_url))
+	print("sending notification to channel: {}".format(slack_channel))
```

Comment:
Please add some log message with clustername which message is about. It will be easier for human to understand what was processed